build.sbt

```diff
@@ -39,8 +37,7 @@ lazy val buildSettings = Seq(
         <email>dev-role@allenai.org</email>
       </developer>
     </developers>,
```

Comment:
what is this: `publishMavenStyle := true,` ? We still have that. Does that require the `pomExtra` stuff? I can try publishing without.